content/en/blog/2024/otel-arrow-production/index.md

```diff
@@ -0,0 +1,420 @@
+---
+title: OpenTelemetry Protocol with Apache Arrow in Production
+linkTitle: OpenTelemetry Arrow in Production
+date: 2024-09-13
```

Comment:
@jmacd @lquerel any preferences for when to publish this? We can do it today or any time this week you prefer

Reply:
I set the date to today.